changes/3333.dependencies

```diff
@@ -0,0 +1 @@
+Update `netutils` to ^1.4.0 from ~1.4.0.
```

Comment:
```suggestion
Updated `netutils` constraint from ~1.4.0 to ^1.4.0 to permit semver upgrades.
```

Reply:
`s/permit semver/permit minor semver/` to @glennmatthews's suggestion